aws/solutions/AmazonCloudWatchAgent/inline/amazon_linux.template

```diff
@@ -131,7 +141,11 @@ Resources:
         # This script below is to install AmazonCloudWatchAgent, restart AmazonCloudWatchAgent and tell the result to cloudformation.
         Fn::Base64: !Sub |
            #!/bin/bash
-           rpm -Uvh https://s3.amazonaws.com/amazoncloudwatch-agent/amazon_linux/amd64/latest/amazon-cloudwatch-agent.rpm
+           curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+           unzip awscliv2.zip
+           sudo ./aws/install
```

Comment:
Isn't the AWS CLI installed by default on Amazon Linux? What is this for?

Reply:
You are right this not needed